### PR TITLE
fetch boost 1.67 if boost is not installed on pc

### PR DESCRIPTION
**Pull request by wenwenchenbosch**

### CODE REVIEW

**Review by SebastianSchildt — APPROVED** [review]
Works. Still working with system boost when installed, and will use a local copy, if not found.

Can be merged @rohoet

**Comment by SebastianSchildt** [comment]
or @daschubert  🍰

File: docker/Dockerfile-Jenkins-Build-Env [hunk 1]
```diff
@@ -4,7 +4,7 @@ FROM ubuntu:18.04
 
 # Install all the pre-requisits we need in the docker image
 RUN apt-get update \
-  && apt-get install -y qemu-user-static docker.io sudo bzip2
+  && apt-get install -y qemu-user-static docker.io sudo bzip2 ccache
```
Comment (rohoet) [inline docker/Dockerfile-Jenkins-Build-Env:7]: Same here. Is `ccache` necessary?

Comment (SebastianSchildt) [reply docker/Dockerfile-Jenkins-Build-Env:7]: https://github.com/eclipse/kuksa.val/pull/42#discussion_r483722967

**Pull request merged**